### DISCUSSION TITLE
Disabled state doesn't work with buttons

### DISCUSSION THREAD

**Discussion by elads11** [TomSchimansky/CustomTkinter#404] [Q&A]
Hey, first of all, great library!
I have a small app for work, and I wanted to stylish it more, so that's how I've come upon this library.
The problem I'm facing is when I'm converting a button for example in my code to your CTKinter button, I get exceptions when I click on it.
That's because it doesn't recognize the state=disabled, which works perfectly with the regular TK button.

**Comment by Akascape** [comment]
@elads11 You have to disable the button like this:

`button.configure(state=tkinter.DISABLED)`

**Reply by elads11** [reply]
Thank you very much!
button.configure instead of
button.config in TK
